lib/storage/index.js

```diff
@@ -132,10 +133,23 @@ async function getSource(paramsSource, inDirectory, disableSourceDownload) {
     return getAsset(paramsSource, inDirectory, name, disableSourceDownload);
 }
 
-async function putRendition(rendition) {
+async function putRendition(rendition, directories) {
     // Note: validation has happened in validate.js before
-
     if (process.env.WORKER_TEST_MODE) {
+        if (rendition.directory === directories.postprocessing) {
```

Comment:
please add "TODO"'s to remind us this is only here for the sdk, not the pipeline so we can remove it when we completely replace the sdk

Reply:
added